samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -164,6 +174,16 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         }
     }
 
+    // Override this to create custom tab counter menu
+    fun createTabCounterMenu(): TabCounterMenu {
+        return TabCounterMenu(requireContext(), onItemTapped())
+    }
+
+    // Override this to create custom tab counter menu
+    fun onItemTapped(): (TabCounterMenu.Item) -> Unit = {
+        // NOOP
+     }
+
```

Comment:
The intention would be that when the user creates their BrowserFragment they define the TabCounterMenu. Is this the right place for this?

Reply:
Yes, I think it's fine via the `BrowserFragment`. We're creating various other menus in A-C/Fenix in `BaseBrowserFragment` and views as well. If we're making the `TabCounterMenu` optional (see above), you actually don't have to create one here for Sample Browser, but it's good for testing :)

Reply:
I think I'll hold off on adding it to the sample browser for now, and add it in as a followup PR 😄